kernel/source/filedesc/calls_inner.c

```diff
@@ -61,6 +61,42 @@ static s32 GetThreadSpecificMsgOrFreeFromExtra(const int useMsgFromExtraInsteadO
 	return IPC_EMAX;
 }
 
+static bool IOSFD_CheckIdValidForProcess(const s32 fd)
```

Comment:
minor change: maybe name this different. a function like `IsFdValidForProcess` or something, as it returns a boolean and this what it does : return if fd is valid :)

Reply:
Alright, but keep the prefix? or remove it because it's a local helper function?

Reply:
if its a local helper function, the scope shows its part of the calls/iosfd functionality. so maybe drop it ye